Sprint-3/alarmclock/alarmclock.js

```diff
@@ -1,4 +1,31 @@
-function setAlarm() {}
+function setAlarm() {
+
+  let alarmInput = parseInt(document.getElementById("alarmSet").value, 10);
+
+  if(isNaN(alarmInput) || alarmInput <= 0) {
+    alert("Please enter a valid time");
```

Comment:
you might want to use something otyher than alert here. Not an issue just not typical :)